cmd/containerd/command/main.go

```diff
@@ -297,6 +300,29 @@ func setLevel(context *cli.Context, config *srvconfig.Config) error {
 	return nil
 }
 
+func setLogFormat(config *srvconfig.Config) error {
+	f := config.Debug.Format
+	if f == "" {
+		f = log.TextFormat
+	}
+
+	switch f {
+	case log.TextFormat:
+		logrus.SetFormatter(&logrus.TextFormatter{
```

Comment:
Do we need to call SetFormatter in this case?

Reply:
I still left it here in case someone wants to be explicit in the config itself.